### ISSUE TITLE
alert script

### ISSUE THREAD

**Issue by tjoozey** [fabien-d/alertify.js#111]
Hello,

This is the code im using to create a simple alert that is prompted automatically when a visitor gives a page hit.

<head><meta charset="utf-8">
<link rel="stylesheet" href="http://fabien-d.github.com/alertify.js/assets/js/lib/alertify/alertify.core.css" />
<link rel="stylesheet" href="http://fabien-d.github.com/alertify.js/assets/js/lib/alertify/alertify.default.css" />

<script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.8.3.min.js"></script>

<script src="http://fabien-d.github.com/alertify.js/assets/js/lib/alertify/alertify.min.js"></script>

<script> $(document).ready(function() { 
    // alert dialog
    alertify.alert("<b>You have reached the WW Network.<Br>The WW Network uses state-of-art<a href="http://www.hasoffers.com/customer-satisfaction/">tracking</a> from our private domain tracker.ww.com.<br><i>We are only accepting<a href="http://network.w-wnetwork.com/signup">*10* new affiliates for January, so join today!</i>");
</script>

Can you tell me how to get this dialog to work? Also, when User hits  the label "ok", how do I make it redirect to a URL?

Awaiting your reply.

**Comment by fabien-d** [comment]
Please don't open new issue when you already have one open. Closing, dup of #110
